source/slang/hlsl.meta.slang

```diff
@@ -20248,3 +20248,23 @@ uint2 clock2x32ARB()
         };
     }
 }
+
+__glsl_version(450)
+__glsl_extension(GL_ARB_shader_clock)
+__glsl_extension(GL_ARB_gpu_shader_int64)
+__target_intrinsic(glsl, clockARB)
```

Comment:
We are deprecating `target_intrinsic` keyword.
Please remove this line and the same line in the other function as well